revscoring/utilities/train_test.py

```diff
@@ -4,20 +4,28 @@
 
 Usage:
     train_test -h | --help
-    train_test <model> [--language=<module>]
-                       [--values-labels=<path>]
-                       [--model-file=<path>]
-                       [--label-type=<type>]
+    train_test <scorer_model> <features> [<language>] [-p=<kv>]...
+               [--version=<vers>]
+               [--values-labels=<path>]
+               [--model-file=<path>]
+               [--label-type=<type>]
 
 Options:
-    -h --help                Prints this documentation
-    <model>                  Classpath to an instance of MLScorerModel to train
-    --values-labels=<path>   Path to a file containing feature values and labels
-                             [default: <stdin>]
-    --model-file=<math>      Path to write a model file to [default: <stdout>]
-    --label-type=<type>      Interprets the labels as the appropriate type
-                             (int, float, str, bool) [default: str]
+    -h --help               Prints this documentation
+    <scorer_model>          Classpath to an the MLScorerModel to construct and
+                            train
+    <features>              Classpath to an list of features to use when
+                            constructing the model
+    -p --parameter=<kv>     A key-value argument pair to use when constructing
+                            the scorer_model.
+    --version=<vers>        A version to associate with the model
```

Comment:
Is this related to https://github.com/halfak/Objective-Revision-Evaluation-Service/issues/24?